### PR TITLE
Make fypp install/export optional

### PR DESCRIPTION
**Pull request by awnawab**
Currently, fckit will override any local fypp if it is included in a bundle. This can be problematic if a project in that bundle needs a newer version of fypp. This PR changes the default behaviour so that fypp is only installed and exported by fckit if it is not already found.
Could you please tag a release once this PR is approved and merged? I would like to include it in an updated ifs-bundle.

### CODE REVIEW

**Comment by FussyDuck** [comment]
[CLA assistant check](https://bol-claassistant.ecmwf.int:443/ecmwf/fckit?pullRequest=35) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://bol-claassistant.ecmwf.int:443/ecmwf/fckit?pullRequest=35) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bol-claassistant.ecmwf.int:443/check/ecmwf/fckit?pullRequest=35) it.</sub>

**Comment by wdeconinck** [comment]
This currently fails the downstream-ci:  
```
/usr/bin/python3.9: can't open file '/opt/actions-runner/work/_work/_temp/install/fckit/libexec/fckit-fypp.py': 
[Errno 2] No such file or directory
```

File: CMakeLists.txt [hunk 1]
```diff
@@ -127,14 +136,18 @@ if( ECBUILD_INSTALL_FORTRAN_MODULES )
...
 file(READ ${CMAKE_CURRENT_SOURCE_DIR}/cmake/add_fctest.cmake ADD_FCTEST)
 file(READ ${CMAKE_CURRENT_SOURCE_DIR}/cmake/fckit_preprocess_fypp.cmake FCKIT_PREPROCESS_FYPP )
 
 set( install_permissions OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE )
 install( FILES tools/fctest-generate-runner.py DESTINATION libexec PERMISSIONS ${install_permissions} )
 install( FILES tools/fckit-eval.sh DESTINATION libexec PERMISSIONS ${install_permissions} )
-install( FILES contrib/fypp-3.0-7895a7e-20200112/bin/fypp DESTINATION libexec RENAME fckit-fypp.py PERMISSIONS ${install_permissions} )
+if( NOT FYPP )
+  install( FILES contrib/fypp-3.0-7895a7e-20200112/bin/fypp DESTINATION libexec RENAME fckit-fypp.py PERMISSIONS ${install_permissions} )
+endif()
```
Comment (wdeconinck) [inline CMakeLists.txt:151]: We need to (re)install the externally found fypp as well, so that downstream packages using fckit's macro can use it as well.

**Review by wdeconinck — CHANGES_REQUESTED** [review]

**Comment by wdeconinck** [comment]
@awnawab version 0.11.1 contains this change.

**Pull request merged**